packages/manager/apps/ips/src/pages/actions/upsertIpBlockInformation/upsertIpBlockInformation.page.tsx

```diff
@@ -0,0 +1,213 @@
+    if (hasOrgChanges && !hasOnGoingChangeRipeOrgTask) changeOrganisation();
+
+    closeModal();
+  };
+
+  useEffect(() => {
+    setNetname(ipRipeInfo?.netname || '');
+    setDescription(ipRipeInfo?.description || '');
+    setOrganisation(organisationId || '');
+  }, [ipRipeInfo, organisationId]);
+
+  useEffect(() => {
+    if (rirForOrganisation === 'ARIN') {
+      setAvailableOrganisations(
+        organisations?.filter(
+          (orgId: string): boolean =>
+            orgId?.toLowerCase().startsWith('arin_') ?? false,
+        ) ?? [],
+      );
+    } else if (rirForOrganisation === 'RIPE') {
+      setAvailableOrganisations(
+        organisations?.filter(
+          (orgId: string): boolean =>
+            orgId?.toLowerCase().startsWith('ripe_') ?? false,
+        ) ?? [],
+      );
+    } else {
+      setAvailableOrganisations(organisations);
+    }
+  }, [organisations, rirForOrganisation]);
+
+  return (
+    <Modal
+      isOpen
+      isLoading={isRipeLoading || isOrgLoading || isOrganisationIdLoading}
+      onDismiss={closeModal}
+      heading={t('ipBlockInformationTitle')}
+      isPrimaryButtonLoading={isRipePending || isOrgPending}
+      primaryLabel={t('validate', { ns: NAMESPACES.ACTIONS })}
+      onPrimaryButtonClick={handleSubmit}
+      primaryButtonTestId="confirm-button"
+      onSecondaryButtonClick={closeModal}
+      secondaryButtonTestId="cancel-button"
+      secondaryLabel={t('cancel', { ns: NAMESPACES.ACTIONS })}
+    >
+      <div className="space-y-4">
+        <div>
+          <OdsText preset="caption" className="mb-2">
+            {t('ipBlockInformationSubtitle')}
+          </OdsText>
+        </div>
+
+        <div>
+          <OdsFormField className="w-full">
+            <label slot="label">{t('ipBlockInformationNetNameLabel')}</label>
+            <OdsInput
+              name="netname"
+              value={netname}
+              onOdsChange={(e) => setNetname(e.detail.value as string)}
+              isDisabled={isRipePending}
+            />
+          </OdsFormField>
+        </div>
+
+        <div>
+          <OdsFormField className="w-full">
+            <label slot="label">
+              {t('ipBlockInformationDescriptionLabel')}
+            </label>
+            <OdsTextarea
+              name="description"
+              value={description}
+              rows={4}
+              onOdsChange={(e) => setDescription(e.detail.value as string)}
+              isDisabled={isRipePending}
+            />
+          </OdsFormField>
+        </div>
+
+        <div>
+          <OdsFormField className="w-full">
+            <label slot="label">
+              {t('ipBlockInformationOrganisationLabel')}
+            </label>
+            <OdsSelect
+              name="organisation"
+              value={organisation}
+              onOdsChange={(e) => setOrganisation(e.detail.value)}
+              isDisabled={isOrgPending || hasOnGoingChangeRipeOrgTask}
```

Comment:
hasOnGoingChangeRipeOrgTask should be used to display a message in my opinion
If you just block the select without any indication the user won't understand